MatrixSDK/Data/MXRoom.m

```diff
@@ -163,6 +163,22 @@ - (MXRoomSummary *)summary
 }
 
 - (void)liveTimeline:(void (^)(MXEventTimeline *))onComplete
+{
+    if ([mxSession.store respondsToSelector:@selector(loadRoomMessagesForRoom:completion:)])
+    {
+        MXWeakify(self);
+        [mxSession.store loadRoomMessagesForRoom:self.roomId completion:^{
```

Comment:
It looks not optimal to not manage it in the queue of the original method (`// Queue the requester`). The queue exists because there can be multiple requests in parallel. It helps to control them.

If it helps, make `loadRoomMessagesForRoom:self` non optional. Having `optional` methods in `MXStore` was not really a good idea.

Reply:
Updated to run in the same queue logic. And method is now non-optional.